app/mod_api/controllers.py

```diff
@@ -171,6 +171,57 @@ def get_metadata(publisher, package):
         return handle_error('GENERIC_ERROR', e.message, 500)
 
 
+@mod_api_blueprint.route("/dataproxy/<publisher>/<package>/<resource>.json", methods=["GET"])
```

Comment:
@zelima my comment is not related to what a client does with data from this endpoint. It is related to the reduced capabilities of the endpoint via hardcoding of file type information, unecessarily I think. It is not the type of thing that should stop merging, but definitely the type of thing to note as an issue in the backlog or something, for the implementing team to keep visibility on it.